codeflash/cli_cmds/cmd_init.py

```diff
@@ -578,8 +578,20 @@ def configure_pyproject_toml(setup_info: SetupInfo) -> None:
     formatter_cmds = []
     if formatter == "black":
         formatter_cmds.append("black $file")
+        print("checking if black exists")
+        try:
+            import black
+        except ImportError as e:
+            print("black not found, installing via pip")
+            subprocess.run(["pip", "install", "black"])
```

Comment:
we should not modify the user environment like this, only recommend the user to install black with this command. same for ruff